src/extras/helpers/ArrowHelper.js

```diff
@@ -94,6 +92,9 @@ THREE.ArrowHelper.prototype.setLength = function ( length, headLength, headWidth
 	if ( headLength < length ){
 		this.line.scale.set( 1, length - headLength, 1 );
```

Comment:
I guess a dot is acceptable here, it should be hidden by the arrow anyway.

Note that I don't really care here, I just want to make that code stop crashing. :)